README.md

```diff
@@ -3,40 +3,49 @@
 Install:
 
 - virtualenv
-- bower
+- docker
+
+We have a collection of tools available under `tools/`. Some are utilities, other are wrappers around other tools to
+make sure that you have the correct versions.
+
+Before running any commands, you have to source our setup script:
+
+    # . .settings.sh
 
 Creating the database:
 
-    # sudo -u postgres psql -f database-setup.sql
+    # cd docker/p2k16
+    # docker-compose up -d
+    # PGUSER=postgres PGPASSWORD=postgres psql -f database-setup.sql
+
+The root user in the database is `postgresql`, it's password is also `postgresql`.
```

Comment:
På linjen over står det `postgres` og her står det `postgresql`, mulig de faktisk er forskjellige, men jeg ble forvirret, så tenkte å spørre om en av de to er feil?

Reply:
Nice catch.